src/syzygy/tbprobe.h

```diff
@@ -30,19 +30,19 @@ class Position;
 namespace Stockfish::Tablebases {
 
 enum WDLScore {
-    WDLLoss        = -2,  // Loss
-    WDLBlessedLoss = -1,  // Loss, but draw under 50-move rule
-    WDLDraw        = 0,   // Draw
-    WDLCursedWin   = 1,   // Win, but draw under 50-move rule
-    WDLWin         = 2,   // Win
+    WDLLoss        = -2,  // A loss
+    WDLBlessedLoss = -1,  // A loss, but a draw under the 50-move rule
+    WDLDraw        = 0,   // A draw
+    WDLCursedWin   = 1,   // A win, but a draw under the 50-move rule
+    WDLWin         = 2,   // A win
 };
 
-// Possible states after a probing operation
+// All possible states after a probing operation
 enum ProbeState {
-    FAIL              = 0,   // Probe failed (missing file table)
-    OK                = 1,   // Probe successful
-    CHANGE_STM        = -1,  // DTZ should check the other side
-    ZEROING_BEST_MOVE = 2    // Best move zeroes DTZ (capture or pawn move)
+    FAIL              = 0,   // The probe has failed (missing a table file)
+    OK                = 1,   // The probe was successful
+    CHANGE_STM        = -1,  // The DTZ should check the other playing side
+    ZEROING_BEST_MOVE = 2    // The best move zeroes the DTZ (a capture or a pawn move)
```

Comment:
I think the above (Line 33 to 45) are better and more concise as they are. Comments are not necessarily meant to be full sentences.

Reply:
These comment changes in tbprobe.h were done in contrast to the corresponding comments in tbprobe.cpp.

What I find inconsistent throughout the whole Stockfish codebase is the fact that some comments are full sentences, while others are not. There is no rule at all as to when a comment needs to be a full sentence and when a concise sentence.

I propose to either have full-sentence comments everywhere in the codebase or have concise ones everywhere. But since the long comments we have (spanning multiple lines) can't really be written in concise form, we should write them in the full-sentence form everywhere.

Maybe you think `meh, I don't care`, but comment form is also important in source code.

Reply:
Stockfish source code doesn’t have to be a novel. Having to write full sentence comments everywhere is way too tedious and annoying.

Reply:
Comments without the `the`, `a` and `an` read very much like broken English.